contracts/DebtRegistry.sol

```diff
@@ -107,6 +107,11 @@ contract DebtRegistry is Pausable {
         _;
     }
 
+    modifier nonNullBeneficiary(address beneficiary) {
```

Comment:
Nit: `beneficiaryNotNull` sounds like less-awkward naming to me, but I won't die on this hill :)

Reply:
I don't like either, so just going to leave as is unless you feel strongly